scripts/lib/helm.sh

```diff
@@ -92,4 +59,21 @@ ensure_controller_pods() {
     echo "Failed to start controller pods. Exiting... "
     TEST_PASS=1
   fi
+}
+
+ensure_helm_chart_installed() {
+  if ! should_execute ensure_helm_chart_installed; then
+     return 1
+  fi
+
+  local __image_tag_suffix="$1"
+  echo "Installing helm chart '$HELM_LOCAL_CHART_NAME' with image $AWS_ECR_REPO_NAME:$__image_tag_suffix"
```

Comment:
probably want to pass the Helm chart name (and the image name) as parameters to the function as opposed to using global Bash variables.

Reply:
yes, earlier we were deploying all controllers together by substituing just version (commit) value.
I am working on changing that to deploying one controller by providing servicealias. Fixing this in next commit.